File: GitToolBox/src/main/java/zielu/intellij/ui/ZUiUtil.java
package zielu.intellij.ui;

import com.intellij.util.ui.UIUtil;
import org.jetbrains.annotations.NotNull;

public class ZUiUtil {
  private ZUiUtil() {
    //do nothing
  }

  @NotNull
  public static String asHtml(@NotNull String content) {
    String contentWithConvertedNewLines = content.replace("\n", "<br>");
    return "<html><head>" + UIUtil.getCssFontDeclaration(UIUtil.getLabelFont()) + "</head><body>"
        + contentWithConvertedNewLines
        + "</body></html>";
  }
}

File: src/main/kotlin/zielu/intellij/ui/ZUiUtil.kt
package zielu.intellij.ui

import com.intellij.util.ui.StartupUiUtil
import com.intellij.util.ui.UIUtil

internal object ZUiUtil {
  @JvmStatic
  fun asHtml(content: String): String {
    val contentWithConvertedNewLines = content.replace("\n", "<br>")
    return (
      "<html><head>" + UIUtil.getCssFontDeclaration(StartupUiUtil.getLabelFont()) + "</head><body>" +
        contentWithConvertedNewLines +
        "</body></html>"
      )
  }
}
